### PR TITLE
update getVolumePath method comment

### PR DESCRIPTION
**Pull request by xigang**
/kind cleanup

**What this PR does / why we need it**:
NONE

**Which issue(s) this PR fixes**:
NONE

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by xigang** [comment]
@vladimirvivien @msau42

File: pkg/hostpath/hostpath.go [hunk 1]
```diff
@@ -160,7 +160,7 @@ func getSnapshotByName(name string) (hostPathSnapshot, error) {
 	return hostPathSnapshot{}, fmt.Errorf("snapshot name %s does not exit in the snapshots list", name)
 }
 
-// getVolumePath returs the canonical path for hostpath volume
+// getVolumePath return the canonical path for hostpath volume
```
Comment (vladimirvivien) [inline pkg/hostpath/hostpath.go]: 👍

Comment (msau42) [reply pkg/hostpath/hostpath.go]: returns

Comment (xigang) [reply pkg/hostpath/hostpath.go]: :-

Comment (pohly) [reply pkg/hostpath/hostpath.go]: @xigang: can you replace "return" with "returns"?

Comment (xigang) [reply pkg/hostpath/hostpath.go]: @pohly Has changed return to returns.

**Pull request merged**